File: ProAgil.Domain/Orador.cs
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Orador
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string MiniCurriculo { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public List<RedeSocial> RedesSociais { get; set; }
        public List<OradorEvento> OradoresEventos { get; set; }
    }
}

File: ProAgil-App/src/app/_models/Orador.ts
import { RedeSocial } from "./RedeSocial";
import { Evento } from "./Evento";

export interface Orador {
    id: number;
    nome: string;
    miniCurriculo: string;
    imagemURL: string;
    telefone: string;
    email: string;
    redesSociais: RedeSocial[];
    oradoresEventos: Evento[];
}
